templates/community_team_list.html

```diff
@@ -18,7 +18,7 @@ <h3>{{ project.name }}</h3>
         {% for member in project.members %}
         <tr>
           <td scope="row">{{ member.role }}</td>
-          <td scope="row">{{ member.name }}</td>
+          <td scope="row" id="{{project.repos}}--{{member.name}}">{{ member.name }}</td>
```

Comment:
There are a few problems with this approach.

- Consider the project Vocabulary which has three repos, so `project.repos` is a comma+space separated string like "fonts, vocabulary, vue-vocabulary". 
- URLs are case-sensitive. So any `member.name` with uppercase characters (basically all names) needs to be typed out exactly as they appear.
- Commas and spaces in the entire URL must be encoded to `%2C` and `%20`.

So an ID like 
```
<... id="fonts, vocabulary, vue-vocabulary--Dhruv Bhanushali">
```
becomes a URL like
```
https://.../#fonts%2C%20vocabulary%2C%20vue-vocabulary--Dhruv%20Bhanushali
```

That's not ideal. How can we solve this? [Jinja filters](https://jinja.palletsprojects.com/en/2.11.x/templates/#list-of-builtin-filters).

- Instead of repos, use `project.name`.
- Lower-case both `project.name` and `member.name` using the [`lower` filter](https://jinja.palletsprojects.com/en/2.11.x/templates/#lower).
- Replace spaces and other non-alphanumeric characters with hyphens using the [`replace` filter](https://jinja.palletsprojects.com/en/2.11.x/templates/#replace).

That should leave us with a very pleasant ID like
```
<... id="vocabulary--dhruv-bhanushali">
```
and a very pleasant URL like
```
https://.../#vocabulary--dhruv-bhanushali
```